js/packages/mobile-wallet-adapter-protocol-web3js/src/transact.ts

```diff
@@ -47,6 +49,24 @@ export async function transact<TReturn>(
             get<TMethodName extends keyof Web3MobileWallet>(target: Web3MobileWallet, p: TMethodName) {
                 if (target[p] == null) {
                     switch (p) {
+                        case 'authorize':
```

Comment:
@steveluscher this is the most complex TS I've ever written. Did I do the thing right?